### PR TITLE
Lastpage Review

### PR DESCRIPTION
**Pull request by TapishKhari**
Lastpage Review:

1. Updated HTML file for the last page.
2. Added images.
3. added css to complete the last page
4. Linter setup and added it
5. Run checks for HTML and css file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -190,6 +190,42 @@ <h3 class="Skill">Skills</h3>
...
+                <div class="last-icons">
+                    <hr class="icons_line">
+                    <ul>
+                        <li><a href="#"><img src="images/Github-black.png" alt="github-icon"></a> </li>
+                        <li><a href="#"><img src="images/linkedin-black.png" alt="linked-icon"></a> </li>
+                        <li><a href="#"><img src="images/chill-black.png" alt="chill-icon"></a> </li>
+                        <li><a href="#"><img src="images/twitter-black.png" alt="twitter-icon"></a> </li>
+                        <li><a href="#"><img src="images/Medium-black.png" alt="Medium-icon"></a> </li>
+                    </ul>
+                    <hr class="final_line">
```
Comment (jasonfelice) [inline index.html:225]: - [ ] **\[OPTIONAL\]** This black bar is not part of the template, it's just there. So, you can consider removing it.
image

File: index.html [hunk 2]
```diff
@@ -190,6 +190,42 @@ <h3 class="Skill">Skills</h3>
             </div>
 
         </section>
+        <footer>
+            <section class="form">
+                <p class="para1">
+            I'm always<br> interested in hearing <br> about new projects, <br> so if you'd like to <br>chat please get in
+                </p>
+                <form action="/action_page.php" class="form_div">
```
Comment (jasonfelice) [inline index.html]: - [ ] Kindly make sure you are using [Formspree service](https://formspree.io/html) to collect the data. Please sign up and use that link.

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @TapishKhari, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Good commit messages✔️
GitHub flow followed✔️
Descriptive PR✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TapishKhari** [comment]
I have added the Formspree in the form action.

**Review by prolajumokeoni — APPROVED** [review]
Hi @TapishKhari 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**